usr/resources/config.rb

```diff
@@ -28,7 +28,7 @@ def self.which_path(cmd)
           '-Dattempt_dir=%{attempt_dir} ' \
           '-Dbasedir=. ' \
           '-l ant.log ' \
-          '-f ../../../../usr/resources/Java/build.xml',
+          '-f ../../../../../usr/resources/Java/build.xml',
```

Comment:
There's an extra `../` in this path. Please double check that these settings are returned to what you see on the `master` branch.